gui/default/syncthing/core/syncthingController.js

```diff
@@ -1097,6 +1097,134 @@ angular.module('syncthing.core')
+                case 'sync-waiting':
+                    return 'fa-hourglass-half';
+                case 'cleaning':
+                    return 'fa-recycle';
+                case 'faileditems':
+                case 'localunencrypted':
+                case 'outofsync':
+                    return 'fa-exclamation-circle';
+                case 'idle':
+                case 'localadditions':
+                    return 'fa-check';
+                case 'paused':
+                    return 'fa-pause';
+                case 'scanning':
+                    return 'fa-search';
+                case 'stopped':
+                    return 'fa-stop';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'unknown':
+                    return 'fa-question-circle';
+                case 'unshared':
+                    return 'fa-unlink';
+            }
+        };
+
+        $scope.folderStatusText = function(folder) {
+            var text = '';
+            switch ($scope.folderStatus(folder)) {
+                case 'clean-waiting':
+                    return $translate.instant('Waiting to Clean');
+                case 'scan-waiting':
+                    return $translate.instant('Waiting to Scan');
+                case 'sync-preparing':
+                    return $translate.instant('Preparing to Sync');
+                case 'sync-waiting':
+                    return $translate.instant('Waiting to Sync');
+                case 'cleaning':
+                    return $translate.instant('Cleaning Versions');
+                case 'faileditems':
+                    return $translate.instant('Failed Items');
+                case 'localunencrypted':
+                    return $translate.instant('Unexpected Items');
+                case 'outofsync':
+                    return $translate.instant('Out of Sync');
+                case 'idle':
+                    return $translate.instant('Up to Date');
+                case 'localadditions':
+                    return $translate.instant('Local Additions');
+                case 'paused':
+                    return $translate.instant('Paused');
+                case 'scanning':
+                    text = $translate.instant('Scanning');
+                    if ($scope.scanPercentage(folder.id) !== undefined) {
+                        text += ' (' +
+                        percentFilter($scope.scanPercentage(folder.id)) +
+                        ')';
+                    }
+                    return text;
+                case 'stopped':
+                    return $translate.instant('Stopped');
+                case 'syncing':
+                    text = $translate.instant('Syncing') +
+                    ' (' + 
+                    percentFilter($scope.syncPercentage(folder.id)) + 
+                    ', ' +
+                    binaryFilter($scope.model[folder.id].needBytes) +
+                    'B)';
```

Comment:
Same as above, and you did have some trailing whitespace here. Maybe teach your editor to remove that automatically?

```suggestion
                    ' (' + percentFilter($scope.syncPercentage(folder.id)) + ', ' +
                    binaryFilter($scope.model[folder.id].needBytes) + ' B)';
```